include/realtime_tools/realtime_publisher.hpp

```diff
@@ -112,6 +124,20 @@ class RealtimePublisher
     }
   }
 
+  /**  \brief Try to get the data lock from realtime and publish the given message
+   *
+   * Tries to gain unique access to msg_ variable. If this succeeds
+   * update the msg_ variable and call unlockAndPublish
+   * @return false in case no lock for the realtime variable could be acquired
+   */
+  bool tryPublish(const MessageT & msg)
+  {
+    if (!trylock()) return false;
+
+    msg_ = msg;
+    unlockAndPublish();
```

Comment:
This is missing a `return true;` I suppose?

Reply:
Good eyes! Thanks for pointing this out.